.github/workflows/tests.yml

```diff
@@ -3,23 +3,63 @@ name: tests
 on:
   push:
   pull_request:
+    branches:
+      - main
+  workflow_dispatch:
   schedule:
-      - cron: '00 4 * * *'  # daily at 4AM
+      - cron: '0 0 * * 0'  # weekly
 
 jobs:
-  build:
+  unit-tests:
 
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
-        python-version: ["3.7", "3.8", "3.9", "3.10"]
+        python-version: ["3.9"] # TODO: expand versions
 
       fail-fast: false
     steps:
 
     - uses: actions/checkout@v4
 
+    # TODO (maffettone): Change host after CD into bluesky/bluesky-pods
+    - name: Download and build bluesky-pods
+      run: |
+        docker pull ghcr.io/maffettone/bluesky-pods-bluesky:latest
+        docker tag ghcr.io/maffettone/bluesky-pods-bluesky:latest bluesky:latest
```

Comment:
Should the upstream images be used here?

`ghcr.io/bluesky/bluesky-pods-bluesky` rather than `ghcr.io/maffettone/bluesky-pods-bluesky`

Reply:
There is no upstream (yet). 
https://github.com/bluesky/bluesky-pods/pull/54